packages/core/src/_common/base/network_monitor_base.js

```diff
@@ -35,9 +35,6 @@ const NetworkMonitorBase = (() => {
                 BinarySocket.close();
                 setNetworkStatus('offline');
             });
-        } else {
-            // default to always online and fallback to WS checks
```

Comment:
what are we doing to trigger WS check in place of this?

Reply:
I don't see a condition where this else block will be triggered. we are listening to online and offline listener in the if condition. Anyways setting navigator.onLine to anything doesn't override its value.